### PR TITLE
#137005173 Static Page Management

### PR DESCRIPTION
**Pull request by andela-jare**
#### What does this PR do?
It adds "Static Pages" management and viewer to the Application.
#### Description of Task to be completed?
1. Added a static pages manager to the dashboard to create and edit static pages.
2. Added a menu to the navbar for clients to view the created static pages.
#### How should this be manually tested?
Via the app on heroku.
#### Any background context you want to provide?
Null.
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/137005173.

### CODE REVIEW

File: imports/plugins/included/static-pages/client/templates/static-pages.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-undef */
```
Comment (andela-aolaniran) [inline imports/plugins/included/static-pages/client/templates/static-pages.js]: Is the disabled eslint rule is necessary ?

File: imports/plugins/included/static-pages/client/templates/static-pages.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import { Template } from "meteor/templating";
+import "./static-pages.html";
+import { StaticPages } from "/lib/collections";
+import { Reaction } from "/client/api";
+
+Template.staticPages.onCreated(() => {
+   // Subscribe to Pages publication
+  Meteor.subscribe("staticPages");
+});
```
Comment (olawalejarvis) [inline imports/plugins/included/static-pages/client/templates/static-pages.js:9]: I think it is better to use subscription Ready method immediately after you subscribe to a particular publication, due to that fact that the result might be needed by your code when it is not ready ---( Javascript synchronous )

File: server/methods/core/static-pages.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import { Meteor } from "meteor/meteor";
+import { check } from "meteor/check";
+import { StaticPages } from  "/lib/collections";
+import * as Schemas from "/lib/collections/schemas";
+
+Meteor.methods({
+  "insertPage"(title, slug, content, shopId, pageOwner) {
```
Comment (olawalejarvis) [inline server/methods/core/static-pages.js:7]: Why don't you follow reaction commerce method declaration style? e.g "insert/page": () => {}

**Review by olawalejarvis — APPROVED** [review]

**Review by victor-enogwe — APPROVED** [review]
Hell Yeah! I Approve.

File: imports/plugins/included/static-pages/client/templates/static-pages.js (hunk 1 above)
Comment (andela-jare) [reply imports/plugins/included/static-pages/client/templates/static-pages.js]: Fixed

File: server/methods/core/static-pages.js (hunk 3 above)
Comment (andela-jare) [reply server/methods/core/static-pages.js:7]: Thanks for the observation but the syntax I used is a valid ES6 syntax and I don't think it is part of RC style guide.

**Review by andela-aolaniran — APPROVED** [review]
Nice job.

**Pull request merged**